src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/SyntaxTokenExtensions.cs

```diff
@@ -223,7 +223,7 @@ public static bool IsBeginningOfGlobalStatementContext(this SyntaxToken token)
                     return true;
 
                 if (token.Parent is FileScopedNamespaceDeclarationSyntax namespaceDeclaration && namespaceDeclaration.SemicolonToken == token)
-                    return true;
+                    return false;
```

Comment:
Since this is changing to `false`, we can remove the condition completely, then the code will fallback to the `return false;` at the end of the method.

But first need to confirm if the new behavior of keyword recommenders is correct.

Reply:
Yep, I don't believe the 'new' keyword should be available directly after the file-scoped namespace declaration anyways. The check doesn't really seem to add anything.